nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -1770,7 +1779,14 @@ N/A
 
     if (*nxagentWindowName == '\0')
     {
-      snprintf(nxagentWindowName, NXAGENTWINDOWNAMELENGTH, "NX");
+      if(nxagentX2go)
+      {
+        snprintf(nxagentWindowName, NXAGENTWINDOWNAMELENGTH, "X2Go Agent");
+      }
+      else
+      {
+        snprintf(nxagentWindowName, NXAGENTWINDOWNAMELENGTH, "NX Agent");
+      }
```

Comment:
Why add "Agent"?

Reply:
> Why add "Agent"?

Because in empty nxagent windows, the "NX" term is IMHO too short and not explaining what that window really is.